stdlib/string.gr

```diff
@@ -1609,3 +1609,66 @@ export let rec forEachCodePointi = (fn: (Number, Number) -> Void, str: String) =
   Memory.decRef(WasmI32.fromGrain(forEachCodePointi))
   void
 }
+
+/**
+ * Trims a string removing any leading or following white spaces based on the end param.
+ *
+ * @param str: The string to be trimmed.
+ * @param end: If true we are trimming the following spaces, otherwise trim leading spaces.
+ * @returns The trimmed string.
+ * 
+ * @example String.trimString("   Hello World   ", false)
+ */
```

Comment:
No graindoc on non-exported member, please. It gives the false impression that docs will be generated for internal functions.